blogs/rtt/index.md

```diff
@@ -0,0 +1,157 @@
+# Network Observability TCP Handshake Round Trip Time
+
+By: Dushyant Behl, Julien Pinsonneau, and Mohamed S. Mahmoud
+
+In OpenShift Container Platform (OCP), ensuring efficient packet delivery is
+paramount for maintaining seamless communication between applications.
+However, challenges like network congestion, misconfigured systems,
+or hardware limitations can lead to slow connections, impacting overall
+performance. Round Trip Time (RTT), typically measured in milliseconds,
+plays a crucial role in monitoring network health and diagnosing issues.
+
+## Implementing Smooth Round-Trip Time (SRTT) with eBPF
+
+The RTT is the time it takes for a packet to travel from the sender to the receiver
+and back. In a network, RTT can vary due to factors like network congestion,
+varying route lengths, and other dynamic conditions.
+SRTT is introduced to provide a more consistent and less jittery representation
+of the RTT.
+
+In TCP, RTT is a crucial metric.
+
+Our implementation leverages eBPF to register to `fentry` eBPF hook
+for `tcp_rcv_established()`.
+We extract the SRTT (smooth round-trip time) value from TCP sockets, correlating it
+to existing flows and enriching them with RTT values in nanoseconds.
+
+When a new Netobserv flow is created, and the RTT (Round-Trip Time) feature is enabled,
+an initial RTT of `10usec` is assigned.
+This initial value for RTT may be considered quite low.
+
+Upon triggering the eBPF (Extended Berkeley Packet Filter) socket, the flow's RTT
```

Comment:
```suggestion
Upon triggering the eBPF (Extended Berkeley Packet Filter) socket, the RTT flow
```

Reply:
instead of making flow possessive, I wonder if we can flip it to "RTT flow value"?